### PR TITLE
Annotations

### PR DESCRIPTION
**Pull request by topepo**
Please edit/add to these for things that need more detail

### CODE REVIEW

File: slides/annotations.qmd [hunk 1]
```diff
@@ -38,9 +43,169 @@ knitr::include_graphics("images/whole-game-final-resamples.svg")
...
+
+If we know a seed value, we can reproduce our "random" numbers. To use a different set of random numbers, choose a different seed value. 
+
+For example: 
+
+```{r}
+set.seed(1)
+runif(3)
+
+# Get a new set or random numbers:
```
Comment (DavisVaughan) [inline slides/annotations.qmd:66]:
```suggestion
# Get a new set of random numbers:
```

File: slides/annotations.qmd [hunk 2]
```diff
@@ -38,9 +43,169 @@ knitr::include_graphics("images/whole-game-final-resamples.svg")
...
+
+# 03 - What  Makes A Model?
+
+## What is wrong with this? 
+
+If we treat the preprocessing as a separate task, it raises the risk that we might accidentally overfit to the data at hand. 
+
+For example, someone might estimate something from the entire data set (such as the principle components) and treat that data as if it were know (and not estimated). Depending on the what was done with the data, consequences in doing that could be:
+
+* You performance metrics are slightly-to-moderately optimistic (e.g. you might think your accuracy us 85% when it is actually 75%)
```
Comment (DavisVaughan) [inline slides/annotations.qmd:95]:
```suggestion
* Your performance metrics are slightly-to-moderately optimistic (e.g. you might think your accuracy is 85% when it is actually 75%)
```

File: slides/annotations.qmd [hunk 3]
```diff
@@ -38,9 +43,169 @@ knitr::include_graphics("images/whole-game-final-resamples.svg")
...
+* A consequential component of the analysis not right and the model just doesn’t work.
```
Comment (DavisVaughan) [inline slides/annotations.qmd:96]:
```suggestion
* A consequential component of the analysis is not right and the model just doesn’t work. 
```

File: slides/annotations.qmd [hunk 4]
```diff
@@ -78,5 +243,81 @@ The way to avoid this problem is to use `step_lincomb(all_numeric_predictors())`
 
 Linear regression detects some redundancies in the predictor set. We can ignore the warnings since `lm()` can deal with it or use [`step_lincomb()`](https://recipes.tidymodels.org/reference/step_lincomb.html) to avoid the warnings. 
 
+## Boosted tree tuning parameters
+
+When the deciding on number of boosting iterations, there are two main strategies:
```
Comment (DavisVaughan) [inline slides/annotations.qmd:248]:
```suggestion
When deciding on the number of boosting iterations, there are two main strategies:
```

**Review by DavisVaughan — APPROVED** [review]

File: slides/annotations.qmd [hunk 5]
```diff
@@ -38,9 +43,169 @@ knitr::include_graphics("images/whole-game-final-resamples.svg")
...
+$$d_{goal} = \sqrt{x^{*2} + y^2}$$ 
+In the code, we log the distance value to help its distribution become more symmetric. 
+
+For angle to center, the formula is 
+
+$$a = \tan^{-1}\left(\frac{y}{x}\right)$$
+This is in radian units and we can convert to degrees using 
+
+$$a = \frac{180}{\pi}\tan^{-1}\left(\frac{y}{x}\right)$$
+For the angle to the goal, we need to alter $x$ and $y$ again. We'll use $x^*$ from above and also use the absolute value of $y$ so that the degrees range from 0 to 180.
```
[the remark this answers is not in the document]
Comment (topepo) [reply slides/annotations.qmd:173]: Line 162. Does not exponentiate since it is within backticks.

**Pull request merged**